pd_xpyd/readme.md

```diff
@@ -0,0 +1,136 @@
+# Guide to Setting Up PD Disaggregation with Mooncake
+
+> **Note**: This document does not cover MLA data parallel setup.
+
+## Mooncake Installation
+
+### Install via pip
+
+```bash
+apt install etcd -y
+pip3 install mooncake-transfer-engine==0.3.0
+```
+
+### Install from Source
+
+```bash
+# Install required packages
+apt install git wget curl net-tools sudo iputils-ping etcd -y
+
+# Clone the Mooncake repository
+git clone https://github.com/kvcache-ai/Mooncake.git -b v0.3.0
```

Comment:
It's duplicated with above mooncake-transfer-engine too.

Reply:
we should keep both pip install and manual build/install step for user.